module/PowerShellEditorServices/Commands/Public/Set-ScriptExtent.ps1

```diff
@@ -5,27 +5,29 @@ function Set-ScriptExtent {
     <#
     .EXTERNALHELP ..\PowerShellEditorServices.Commands-help.xml
     #>
-    [CmdletBinding(PositionalBinding=$false, DefaultParameterSetName='__AllParameterSets')]
+    [CmdletBinding(PositionalBinding = $false, DefaultParameterSetName = '__AllParameterSets')]
     param(
-        [Parameter(Position=0, Mandatory)]
-        [psobject]
-        $Text,
+        [Parameter(Position = 0, Mandatory)]
+        [psobject] $Text,
 
-        [Parameter(Mandatory, ParameterSetName='AsString')]
+        [Parameter(Mandatory, ParameterSetName = 'AsString')]
         [switch]
         $AsString,
```

Comment:
Is it intentional that `$AsString,` is still kept on its own line here?

Reply:
Nah not intentional, just missed it